transformer/transformer_test.go

```diff
@@ -309,8 +309,8 @@ func TestCustom(t *testing.T) {
 }
 
 func TestCustomFail(t *testing.T) {
-	r := rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
-	if html, _, err := Process(&r); err == nil {
+	r := &rpb.Request{Html: "<html ⚡><lemur>", Config: rpb.Request_CUSTOM, Transformers: []string{"does_not_exist"}}
```

Comment:
OOC, what was the error/warning that precipitated this change?

Reply:
transformer/transformer_test.go:314:49: call of t.Fatalf copies lock value: github.com/ampproject/amppackager/transforme
r/request.Request contains google.golang.org/protobuf/internal/impl.MessageState contains sync.Mutex

Reply:
Ah, huh, good to know, thx.